stdlib/public/libexec/swift-backtrace/main.swift

```diff
@@ -611,7 +652,13 @@ Generate a backtrace for the parent process.
       description = "Program crashed: \(target.signalDescription) at \(hex(target.faultAddress))"
     }
 
-    writeln("")
+    // Clear the message written by the crash handler
+    if args.color {
+      write("\r\u{1b}[0K")
+    } else {
+      write("\r" + String(repeating:" ", count: 80) + "\r")
```

Comment:
This hardcoded `80` is unfortunate, but it looks like this is just a fallback, so probably not a big deal?

Reply:
Yeah, there isn't much we can do if we want to erase the message that was there before. We could choose instead to not erase it, and make the message different, maybe. Perhaps that would be OK for non-color mode?

Reply:
I _suppose_ we could work out how long the message actually was, but if it's over the width of the terminal then you end up with odd output (since you get a line wrap).

Reply:
That seems reasonable. I assume non-color mode is rare, outside of having your output piped somewhere. And if it's piped, trying to clear the message will just add garbage.

Reply:
I fixed it so that we print a separate message in the non-color case, and also modified the code that deals with what happens when we don't find `swift-backtrace` so that _if you explicitly enable backtracing_, you'll get a warning that it's been disabled because we didn't have `swift-backtrace`.